subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,28 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
+        cid = self.get_cid()
+        cont_long_id = self.sub_stuff['cont'].json_by_long_id(cid)
```

Comment:
Yes, either of those would be better.

I realize this is a fine point but it's an important one: our job as programmers is to write code that someone, some day, will find easy to understand. Good variable and function names will help that future person.